Sources/BraintreeUIKit/Components/BTUIKFormField.m

```diff
@@ -78,26 +79,20 @@ - (void)updateConstraints {
     
     BOOL hasFormLabel = (self.formLabel.text.length > 0);
     
-    [self.layoutConstraints addObjectsFromArray:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|[textField]|"
+    [self.layoutConstraints addObjectsFromArray:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-(PADDING)-[textField]-(PADDING)-|"
                                                                  options:0
                                                                  metrics:metrics
                                                                    views:viewBindings]];
 
-    [self.layoutConstraints addObjectsFromArray:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|[formLabel]|"
+    [self.layoutConstraints addObjectsFromArray:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-(PADDING)-[formLabel]-(PADDING)-|"
```

Comment:
Since we can't use a fixed height for these form fields anymore, it made sense to add in a little padding above and below so that things don't look too squished.